jme3-core/src/main/java/com/jme3/renderer/opengl/GLRenderer.java

```diff
@@ -630,6 +631,16 @@ public void initialize() {
                 gl2.glEnable(GL2.GL_POINT_SPRITE);
             }
         }
+
+	IntBuffer tmp=BufferUtils.createIntBuffer(16);
+	gl.glGetInteger(GL.GL_FRAMEBUFFER_BINDING, tmp);
+	tmp.rewind();
+	int fbOnLoad=tmp.get();
+	if(fbOnLoad>0)
+	{
+            // Overriding default FB to fbOnLoad. Mostly iOS fix for scene processors and filters
+	    defaultFBO=fbOnLoad;
+	}
```

Comment:
Can you please fix formatting issues here? 
(spacing before and after '=', '>' and in 'if('  )

Reply:
Done ;)

Reply:
Thanks